packages/v4/src/content/design-guidelines/usage-and-behavior/wizard/wizard.md

```diff
@@ -122,13 +121,13 @@ A progressive wizard takes the same form as the standard wizard or wizard with s
 
 Here is an example...
 
-![progressive wizard step 1](./img/wizard-progressive-step1.png)
+<img src="./img/wizard-progressive-step1.png" alt="Wizard progressive step 1" width="990"/>
 Step 1: The user is presented with a Get started screen where they can specify what they want to do.
 
-![progressive wizard step 2](./img/wizard-progressive-step2-new.png)
+<img src="./img/wizard-progressive-step2-new.png" alt="Wizard progressive step 2" width="990"/>
```

Comment:
Same thing here - need to add a line break, by just adding an extra line of space between the image and the text